### PR TITLE
Add a move element command to Display Test App for testing.

### PR DESCRIPTION
**Pull request by MarcNeely**
Added a "dta move element" command to display test app to test the moving of elements when editing.

### CODE REVIEW

File: test-apps/display-test-app/src/frontend/EditingTools.ts [hunk 1]
```diff
@@ -253,3 +253,68 @@ export class PlaceLineStringTool extends CreateElementTool {
...
+ * @alpha
+ */
+export class MoveElementTool extends Tool {
+  public static override toolId = "MoveElement";
+  public static override get minArgs() { return 1; }
+  public static override get maxArgs() { return 4; }
+
+  /** This method runs the tool, adding a reality model to the viewport
+   * @param url the URL which points to the reality model tileset
+   */
```
Comment (pmconne) [inline test-apps/display-test-app/src/frontend/EditingTools.ts:287]:
```suggestion
```
Comment not relevant to this tool.

File: test-apps/display-test-app/src/frontend/EditingTools.ts [hunk 2]
```diff
@@ -253,3 +253,68 @@ export class PlaceLineStringTool extends CreateElementTool {
...
+}
+
+export async function transformElements(imodel: BriefcaseConnection, ids: string[], transform: Transform) {
+  await startCommand(imodel);
+  await basicManipulationIpc.transformPlacement(compressIds(ids), transform.toJSON());
+}
+
+/** This tool adds a seequent reality model to the viewport.
+ * @alpha
+ */
```
Comment (pmconne) [inline test-apps/display-test-app/src/frontend/EditingTools.ts:279]:
```suggestion
```

File: test-apps/display-test-app/src/frontend/EditingTools.ts [hunk 3]
```diff
@@ -253,3 +253,68 @@ export class PlaceLineStringTool extends CreateElementTool {
...
+    return true;
+  }
+
+  /** Executes this tool's run method with args[0] containing the `url` argument.
+   * @see [[run]]
+   */
+  public override async parseAndRun(...args: string[]): Promise<boolean> {
+    let x = 0;
+    let y = 0;
+    let z = 1;
```
Comment (pmconne) [inline test-apps/display-test-app/src/frontend/EditingTools.ts:309]: 1 is desired default for z, so `dta move element 0x123 1` will offset by (1, 0, 1)?

File: test-apps/display-test-app/src/frontend/EditingTools.ts [hunk 4]
```diff
@@ -253,3 +253,68 @@ export class PlaceLineStringTool extends CreateElementTool {
...
+
+function orderIds(elementIds: string[]): OrderedId64Array {
+  const ids = new OrderedId64Array();
+  elementIds.forEach((id) => ids.insert(id));
+  return ids;
+}
+
+function compressIds(elementIds: string[]): CompressedId64Set {
+  const ids = orderIds(elementIds);
+  return CompressedId64Set.compressIds(ids);
```
Comment (pmconne) [inline test-apps/display-test-app/src/frontend/EditingTools.ts:265]:
```suggestion
function compressIds(elementIds: string[]): CompressedId64Set {
  return CompressedId64Set.sortAndCompressIds(elementIds);
```

**Review by pmconne — APPROVED** [review]

File: test-apps/display-test-app/src/frontend/EditingTools.ts (hunk 1 above)
Comment (MarcNeely) [reply test-apps/display-test-app/src/frontend/EditingTools.ts]: done

Comment (MarcNeely) [reply test-apps/display-test-app/src/frontend/EditingTools.ts]: done

File: test-apps/display-test-app/src/frontend/EditingTools.ts (hunk 2 above)
Comment (MarcNeely) [reply test-apps/display-test-app/src/frontend/EditingTools.ts]: done

File: test-apps/display-test-app/src/frontend/EditingTools.ts (hunk 3 above)
Comment (MarcNeely) [reply test-apps/display-test-app/src/frontend/EditingTools.ts]: changed default behavior

**Pull request merged**